templates/partGun_GSD_template.py

```diff
@@ -78,3 +78,24 @@
         psethack = cms.string('single or multiple particles predefined E moving vertex'),
         firstRun = cms.untracked.uint32(1)
     )
+elif 'physproc' in gunmode:
+
+    #physproc should be a string  of the form physproc:proc[:jetColl:threshold:min_jets]
+
+    proc_cfg=gunmode.split(':')[1:]    
+    proc=proc_cfg[0]
+
+    from reco_prodtools.templates.hgcBiasedGenProcesses_cfi import *
+
+    #define the process
+    print 'Setting process to',proc
```

Comment:
I think it would be useful if in addition the actual process also the other configurable parameters (that are used below) were printed at this stage.